controller/table.mustache

```diff
@@ -3,15 +3,12 @@ const r = require('../lib/db.js');
 module.exports = () => {
   return r.tableCreate('{{pluralName}}').run()
   .catch((err) => {
-    if (err.message.split('\n')[0] === 'Table `'+process.env.RETHINK_NAME+'.{{pluralName}}` already exists in:') return;
+    let errMsg = `Table \`${process.env.RETHINK_NAME}`;
+    errMsg += `.{{pluralName}}\` already exists in:`;
+    if (err.message.split('\n')[0] === errMsg) return;
     throw err;
   })
   .then(() => {
-    r.table('{{pluralName}}').wait().run()
-  })
-  //.then(() => r.table('{{pluralName}}').indexCreate('name').run())
-  //.catch(err => {
-  //  if (err.message.indexOf('Index `name` already exists') > -1 )return;
-  //  throw err;
-  //})
-}
```

Comment:
This needs to remain in place. It's to show users of the scaffold how they can add indexes to a table rather than having to look up the docs every time.